src/content/development/release-process/_index.en.md

```diff
@@ -92,7 +99,7 @@ You can run the process without pushing the PR automatically (obviating the need
 To do so, run the `make` command with `dryrun=true`.
 {{% /notice %}}
 
-The command runs, gathers the data for the release, updates the release YAML and pushes it for review. Once the review process is done,
+The command runs, gathers the data for the release and updates the release YAML and pushes it for review. Once the review process is done,
```

Comment:
I am not sure why this change is required? The existing text looks good to me.

Reply:
It's a mistake my IDE has auto-correction enabled. Thanks :+1: